include/ibamr/IBImplicitStaggeredHierarchyIntegrator.h

```diff
@@ -150,11 +156,113 @@ class IBImplicitStaggeredHierarchyIntegrator : public IBHierarchyIntegrator
 
 private:
     /*!
-     * \brief Default constructor.
-     *
-     * \note This constructor is not implemented and should not be used.
-     */
-    IBImplicitStaggeredHierarchyIntegrator();
+	 * \brief A StaggeredStokesSolver that does nothing.
```

Comment:
Yeah, the SNES function works with inhomogeneous BC incorporated into Stokes Op, so we don't need to worry about subtracting inhomogeneous BCs from RHS.